WeWatch/WeWatch/Custom TabBar/TabBarViewModel.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  TabBarViewModel.swift
+//  WeWatch
+//
+//  Created by Anton on 30/12/2024.
+//
+
+import Foundation
+import SwiftUI
+
+internal final class TabBarViewModel: ObservableObject {
+    
+    @Published internal var homeViewIsActive: Bool = true
+    @Published internal var searchViewIsActive: Bool = false
+    @Published internal var bookmarkViewIsActive: Bool = false
```

Comment:
There is no need to have these properties if you decided to use `TabViewType` enum. So you can easily remove them